vigilate_backend/views.py

```diff
@@ -258,16 +260,17 @@ def get_queryset(self):
         else:
             return Station.objects.filter(user=self.request.user.id)
 
-class SessionViewSet(viewsets.ModelViewSet):
+class SessionViewSet(viewsets.mixins.CreateModelMixin,
```

Comment:
By specifiying the allowed mixins, we don't have the problem of allowing list() without allowing retrieve() and the OPTIONS are not blocked because of the default permission_classes used is IsAuthenticatedOrOptions.